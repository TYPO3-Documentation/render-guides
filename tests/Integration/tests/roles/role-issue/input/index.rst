==============
Document Title
==============

Lorem Ipsum Dolor. See also :issue:`102056` or :issue:`Lorem Ipsum <99508>`.
