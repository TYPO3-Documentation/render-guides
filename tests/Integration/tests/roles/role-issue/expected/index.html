<!-- content start -->
                <section class="section" id="document-title">
    <h1>Document Title<a class="headerlink" href="#document-title" title="Permalink to this headline">Â¶</a></h1>

            <p>Lorem Ipsum Dolor. See also <a href="https://forge.typo3.org/issues/102056">forge#102056</a> or <a href="https://forge.typo3.org/issues/99508">Lorem Ipsum</a>.</p>
    </section>

        <!-- content end -->
